24 add objective magnification factor

StroemPhi opened the pull request:
This PR closes #24 by adding the new class "objective magnification factor setting" within the TSV template and updating the component OWl generated from it.

Zack-83 commented on src/templates/vibso_terms.tsv:
```diff
@@ -14,3 +14,4 @@ obo:VIBSO_0000010	Raman spectrometer configuration	A device setting that represe
 obo:VIBSO_0000011	excitation wavelength	A physical quality of the light source that is used used to excite a system to a higher energy state.	DOI:10.1351/goldbook	owl:Class	wavelength					ORCID:0000-0002-1595-3213			This class needs to be defined in PATO or another more general ontology for such qualities.
 obo:VIBSO_0000012	diffraction grating	A device that is an optical component with a periodic structure that diffracts light into several beams travelling in different directions (i.e. different diffraction angles)	https://en.wikipedia.org/wiki/Diffraction_grating	owl:Class	device					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontol ...
 obo:VIBSO_0000013	excitation wavelength setting	A setting datum that specifies the wavelength of an electromagnetic radiation source (e.g. a laser) which has the function to excite the energy state of a target material.		owl:Class	setting datum					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devic ...
+obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0002-1595-3213		:experimentXY obo:VIBSO_0000014 '100x' .	
```
```suggestion
obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0001-7694-5519; ORCID:0000-0002-2239-3955		:experimentXY obo:VIBSO_0000014 '100x' .	
```

StroemPhi replied:
done in [4562d5a](https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/pull/25/commits/4562d5ad60888e9f7382f9c8366b4d54e65021e9)

StroemPhi commented:
Hi @Zack-83, I have added you two as term creators and updated the example as a TTL snippet the way it should be modeled. 
In your example suggestion you used this class incorrectly as the predicate (as data property) :wink:
In the example, I took the shortcut of using the data property OBI:'has representation' attached to the instance of this kind of setting instead of using a OBI:value specification instance related to it via OBI:'has value specification'.
I hope this example is helpful for you

Zack-83 commented on src/templates/vibso_terms.tsv:
```diff
@@ -14,3 +14,12 @@ obo:VIBSO_0000010	Raman spectrometer configuration	A device setting that represe
 obo:VIBSO_0000011	excitation wavelength	A physical quality of the light source that is used used to excite a system to a higher energy state.	DOI:10.1351/goldbook	owl:Class	wavelength					ORCID:0000-0002-1595-3213			This class needs to be defined in PATO or another more general ontology for such qualities.
 obo:VIBSO_0000012	diffraction grating	A device that is an optical component with a periodic structure that diffracts light into several beams travelling in different directions (i.e. different diffraction angles)	https://en.wikipedia.org/wiki/Diffraction_grating	owl:Class	device					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontol ...
 obo:VIBSO_0000013	excitation wavelength setting	A setting datum that specifies the wavelength of an electromagnetic radiation source (e.g. a laser) which has the function to excite the energy state of a target material.		owl:Class	setting datum					ORCID:0000-0002-1595-3213			This class should better be defined in a more general device ontology or a domain ontology with the scope of optical devic ...
+obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0002-1595-3213;ORCID:0000-0001-7694-5519;ORCID:0000-0002-2239-3955		"@prefix ex: <https://example.org/> .
+@prefix obo: <http://purl.obolibrary.org/obo/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+
+ex:_RamanSpectrometerX rdf:type obo:CHMO_0001235 (Raman spectrometer) ;
+            obo:OBI_0000293 (has_specified_input) ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX .
+ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX obo:VIBSO_0000014 ;
+            obo:OBI_0002815 (has representation) ""100x"" ."	
```
Well done. I suggest keeping the examples as short as possible. So: no prefix definition, short names.
Long strings with quotes should be reported with triple double quotes. (We might also alternate single and double quotes.) 

```suggestion
obo:VIBSO_0000014	objective magnification factor setting	A setting datum that specifies the magnification factor of the objective collecting the scattered light.		owl:Class	setting datum					ORCID:0000-0002-1595-3213;ORCID:0000-0001-7694-5519;ORCID:0000-0002-2239-3955		"""ex:_RamanSpectrometerX rdf:type obo:CHMO_0001235 (Raman spectrometer) ;
            obo:OBI_0000293 (has_specified_input) ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX .
ex:_objective_magnification_factor_setting_for_RamanSpectroscopeX rdf:type obo:VIBSO_0000014 ;
            obo:OBI_0002815 (has representation) "100x" ."""	
```

StroemPhi replied:
@Zack-83 thanks for spotting the missing rdf:type. I didn't do the masking with single quote, Libre Office Calc did this by default. (I edited the TSV in there.) I'm also fine with using tripple quotes, but the way I had it before works with the ROBOT conversion step which produces the OWL from the TSV without then having the whole string block in quotes in the owl.

Zack-83 reviewed (changes requested):
Some changes in the example (within the TSV).

StroemPhi reviewed (commented):
All good now.

StroemPhi commented:
@Zack-83 I just noticed that in order to display the TSV as a table in GitHub, we cannot use tripple quotes but must use single quotes. See: https://github.com/NFDI4Chem/VibrationalSpectroscopyOntology/blob/StroemPhi-patch-1/src/templates/vibso_classes.tsv

Zack-83 commented:
It is deadly difficult to correct a TSV table without forgetting a tab.

And the poor English knowledge of GitHub does not help :(

![image](https://user-images.githubusercontent.com/37865804/210568074-cbba22ff-9ba2-4089-a3cf-eea4591c2d0b.png)

Just merge yours and then it's done.
